src/gui/dialogs/unit_recall.cpp

```diff
@@ -248,7 +249,7 @@ void unit_recall::pre_show(window& window)
 
 	list.register_sorting_option(0, [this](const int i) { return recall_list_[i]->type_name().str(); });
 	list.register_sorting_option(1, [this](const int i) { return recall_list_[i]->name().str(); });
-	list.register_sorting_option(2, [this](const int i) { return recall_list_[i]->level(); });
+    list.register_sorting_option(2, [this](const int i) { return recall_list_[i]->level()*1000 - recall_list_[i]->experience_to_advance(); });
```

Comment:
You changed the line to be indented by spaces. Please change it back to tab indentation.